### DISCUSSION TITLE
Goto definition not working for svelte lsp in termux

### DISCUSSION THREAD

**Discussion by mgholam** [LazyVim/LazyVim#4897] [Q&A]
Hi all,

I have `lavyvim` installed in `termux` with `svelte-language-server` and the "goto definition"  does not work, while the same works fine for my `golang` lsp, any clues?

**Comment by darcyparker** [comment]
I have same issue with `vtsls`.  goto definition works inside same file, but not across files. This used to work before new year. (I had v14 installed before new year too.)

**Reply by darcyparker** [reply]
`:LspInfo` wasn't reporting any servers attached to the current buffer.  I had to run `:LspStart`.  Strange. The server used to start automatically before the new year.  And `:LspInfo` says `vtsls`'s autostart is true... But it is not auto starting.

**Comment by mgholam** [comment]
The problem can be fixed by using `termux-chroot` which fixes the root path.

**Reply by darcyparker** [reply]
I think my issue is different then... My goto definition wasn't working because the lsp server wasn't started. (autostart is true, but for some reason I have to manually start it.)

**Comment by deejinng** [comment]
How do you all even pass this level?
  <br>                                                                                                                                         `2 luarocks                                                                                                                                    3 - checking hererocks installation                                                                                                           4 - ✅ OK no plugins require luarocks, so you can ignore any warnings below                                                                   5 - ✅ OK {python3} Python 3.12.11                                                                                                            6 - ❌ ERROR {/data/data/com.termux/files/home/.local/share/nvim/lazy-rocks/hererocks/bin/luarocks} not installed                             7 - ⚠️  WARNING {/data/data/com.termux/files/home/.local/share/nvim/lazy-rocks/hererocks/bin/lua} version 5.1 not installed                    8 - ⚠️  WARNING Lazy won't be able to install plugins that require luarocks.                                                                   9   Here's what you can do:                                                                                                                  10    - fix your luarocks installation                                                                                                        11    - disable hererocks with opts.rocks.hererocks = false                                                                                   12    - disable luarocks support completely with opts.rocks.enabled = false`
